File: vademecum/code-src/podstawy/Skoki_warunki_pętle_funkcje.js

var i, j, k;

// instrukcja waunkowa if - else
if (i<j) {
	console.log("i<j\n");
} else if (j<k) {
	console.log("i>=j AND j<k\n");
} else {
	console.log("i>=j AND j>=k\n");
}

// podstawowe operatory logiczne
if (i<j || j<k)
	console.log("i<j OR j<k\n");
// innymi operatorami logicznymi są && (AND), ! (NOT)

// pętla for
for (i=2; i<=9; ++i) {
	if (i==3) {
		// pominięcie tego kroku pętli
		continue;
	}
	if (i==7) {
		// wyjście z pętli
		break;
	}
	console.log(" a: ", + i + "\n");
}

// pętla while
while (i>0) {
	console.log(" b: " + --i + "\n");
}

// pętla do - while
do {
	console.log(" c: " + ++i + "\n");
} while (i<2);

// instrukcja wyboru switch
switch(i) {
	case 1:
		console.log("i==1\n");
		break;
	default:
		console.log("i!=1\n");
		break;
}

File: vademecum/code-src/podstawy/Skoki_warunki_pętle_funkcje.py

i=k=j=0

# instrukcja waunkowa if - else
if i<j :
	print("i<j")
elif j<k :
	print("i>=j AND j<k")
else:
	print("i>=j AND j>=k")

# podstawowe operatory logiczne
if (i<j or j<k)
	puts("i<j OR j<k");
# innymi operatorami logicznymi są and oraz not


# pętla for
for i in range(2, 9):
	if i==3:
		# pominięcie tego kroku pętli
		continue;
	if i==7:
		# wyjście z pętli
		break;
	print(" a:", i);
else:
	# nie wejdzie tutaj bo pętla kończy
	# się z użyciem break
	print("else w pętli");

# pętla while
while i>0 :
	i = i - 1;
	print(" b:", i);
else:
	# wejdzie tu gdy warunek i>0
	# nie będzie spełniony
	# zarówno przy pierwszym
	# jak i kolejnych sprawdzeniach
	print("else w pętli");
